exercise_perseus_render/assets/src/vue/perseus.vue

```diff
@@ -457,6 +464,9 @@
     .paragraph
       padding: 4px
 
+    .fixed-to-responsive
+      display: inline-block
+
```

Comment:
Yeah, unfortunately I don't think it's possible to apply our own styling to Perseus in a safe way. It's definitely not going to look as nice as it does now because the colors and fonts will be inconsistent with Kolibri. The content renderer will be more like a "viewport" into another app than an integrated experience with Kolibri.